test/unit/foundation.test.ts

```diff
@@ -131,4 +133,30 @@ describe('foundation', () => {
     it('does not render empty objects into its template', () =>
       expect(empty).dom.to.be.empty);
   });
+
+  describe('getDataSink', () => {
+    const doc = getDocument();
+    it('returns an array of ExtRef connected to the FCDA', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7203429)
```suggestion
    it("returns an array of ExtRef connected to the FCDA", () => {
```